sdk/communication/Azure.Communication.CallAutomation/src/CallMedia.cs

```diff
@@ -467,8 +467,9 @@ public virtual async Task<Response<HoldResult>> UnholdAsync(UnholdOptions option
             try
             {
                 var request = new UnholdRequestInternal(CommunicationIdentifierSerializer.Serialize(options.TargetParticipant));
+                request.OperationContext = options.OperationContext;
```

Comment:
*NIT*: `UnholdRequestInternal` has constructor for `operationContext`
```suggestion
 new UnholdRequestInternal(CommunicationIdentifierSerializer.Serialize(options.TargetParticipant), options.OperationContext);
```